packages/mobile/src/identity/verification.ts

```diff
@@ -549,52 +626,83 @@ function* isCodeAlreadyAccepted(code: string) {
   return existingCodes.find((c) => c.code === code)
 }
 
-function* revealNeededAttestations(
+function* revealAttestations(
   attestationsWrapper: AttestationsWrapper,
   account: string,
   phoneHashDetails: PhoneNumberHashDetails,
-  attestations: ActionableAttestation[],
-  withoutRevealing: boolean = false
+  attestations: ActionableAttestation[]
 ) {
-  Logger.debug(TAG + '@revealNeededAttestations', `Revealing ${attestations.length} attestations`)
-  const delayPeriod = 5000
+  Logger.debug(TAG + '@revealAttestations', `Revealing ${attestations.length} attestations`)
   let i = 0
+  const delayPeriod = 5000
+  const reveals = []
+  for (const attestation of attestations) {
+    const result = yield call(
+      revealAttestation,
+      attestationsWrapper,
+      account,
+      phoneHashDetails,
+      attestation,
+      // TODO (i1skn): remove this method and uncomment revealNeededAttestations above
+      // when https://github.com/celo-org/celo-labs/issues/578 is resolved
+      // send messages with 5000ms delay on Android
+      Platform.OS === 'android' ? delayPeriod * i : 0
```

Comment:
Isn't it more simple to simply add a `delay` or `sleep` call instead of passing an extra `delayInMs` parameter? That would simplify the `revealAttestation` generator, eliminate the need for `i`, and make it really simple to eliminate the delay when we no longer need it